Drone/drone_base.py

```diff
@@ -0,0 +1,361 @@
+# Standard Library
+import abc
+import coloredlogs
+import dronekit
+from dronekit import VehicleMode
+import logging
+import math
+import time
+import threading
+
+# Ours
+from ..Utilities import constants as c
+from ..Utilities import dronekit_wrappers as dkw
```

Comment:
Consider using a slightly more descriptive name than dkw, it could be confusing to developers whom are not as familiar with the code.

Reply:
Either that or establish the standard of always importing the module by that name.

Reply:
I have changed it so that it uses the full name.